### PR TITLE
Update skipping-workflow-runs.md

### LINKED ISSUE #18434: Page 'Skip workflow runs' - Difference between two code samples not visible due to wrapping.
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[GitHub Actions / Managing workflow runs / Skip workflow runs](https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs)
----
image
----
As display in
**Chrome Version 102.0.5005.63 (Official Build) (64-bit)**
on **Windows 10 Pro**

### What changes are you suggesting?

Swapping the two code parts, would put the one with the space character entirely on the next line. This way, the main difference between these code parts, the space character, is visually preserved.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by vHeemstra**
### Why:

The first code tag will wrap the ` true` part to the next line, hiding the only difference (the space character).

Closes #18434 

### What's being changed:

Swap these two should put the one with the space to next line and thus showing the space character.

**Before:**
image

**After:**
image

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@vHeemstra Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/actions/managing-workflow-runs/skipping-workflow-runs.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Workflows that would otherwise be triggered using `on: push` or `on: pull_reques
 * `[skip actions]`
 * `[actions skip]`
 
-Alternatively, you can end the commit message with two empty lines followed by either `skip-checks: true` or `skip-checks:true`.
+Alternatively, you can end the commit message with two empty lines followed by either `skip-checks:true` or `skip-checks: true`.
```
Comment (martin389) [inline content/actions/managing-workflow-runs/skipping-workflow-runs.md:29]: This approach might be more broadly compatible with different window sizes:
```suggestion
Alternatively, you can end the commit message with two empty lines followed by either:
- `skip-checks:true`
- `skip-checks: true`
```

**Comment by martin389** [comment]
Thanks for this PR! I've just compared with my browser and its rendering differently for me:
image I've added a suggestion with a different approach to this.

**Comment by vHeemstra** [comment]
Done 😄

**Comment by martin389** [comment]
Looks good in staging ✨ 

image

**Review by martin389 — APPROVED** [review]

**Comment by martin389** [comment]
Thanks for the PR! 👍 ✨

**Pull request merged**